Multiple repository url (#1483)
kyma-project/kyma: components/helm-broker/README.md

## Development

Before each commit, use the `before-commit.sh` script, which tests your changes.

### Run a local version

To run the application without building the binary file, run this command:

```bash
APP_KUBECONFIG_PATH=/Users/$User/.kube/config APP_CONFIG_FILE_NAME=contrib/minimal-config.yaml APP_REPOSITORY_URLS=https://github.com/kyma-project/bundles/releases/download/0.1.0/ APP_CLUSTER_SERVICE_BROKER_NAME=core-helm-broker APP_HELM_BROKER_URL=http://localhost:8080 go run cmd/broker/main.go
```

> Be aware when you run the Helm Broker locally you cannot test all features. Only part of them are working correctly.

Author: added a comment

> Be aware when you run the Helm Broker locally you cannot test all features. Only part of them are working correctly. All features which performs an action with Tiller won't work.

Reviewer: ```suggestion
>**NOTE:**  Not all features are available when you run the Helm Broker locally. All features which perform actions with Tiller do not work.
```
